app/static/src/scss/record.scss

```diff
@@ -139,6 +139,10 @@
         width:0;
         text-align:right;
         font-size: 1rem;
+
+        &__left {
```

Comment:
needed to left align for my table - didnt want to break all the others if they need to be right aligned for some reason